src/test/incremental/thinlto/cgu_keeps_identical_fn.rs

```diff
@@ -12,7 +12,7 @@
 
 #![rustc_expected_cgu_reuse(module="cgu_keeps_identical_fn-foo",
                             cfg="cfail2",
-                            kind="no")]
+                            kind="post-lto")]
```

Comment:
What does this change mean?

Reply:
This change means that we reuse a CGU instead of redoing codegen from scratch on it.